Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -471,7 +471,9 @@ class TrackStateProxy {
     typeFlags() = other.typeFlags();
 
     // can be nullptr, but we just take that
```

Comment:
I guess this comment isn't really valid any more?

Reply:
Good catch, thanks!